### PR TITLE
US 025: Create GET operation to obtain histories

### PR DESCRIPTION
**Pull request by RobertoPloneda**

### CODE REVIEW

File: src/test/resources/GET.feature [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
     And I am targeting "at-sce-api" service
+    And I have access to "at-sce-db" database
 
 
-@US_001 @1
-  Scenario: Get all users
-  Given I want to retrieve all users
-  When I send a GET request
-  Then the status code should be "200"
-  And information retrieved from service should match with DB
```
Comment (ramiroAT) [inline src/test/resources/GET.feature:15]: Why did you delete this scenario? Is it no longer used or needed?

Comment (RobertoPloneda) [reply src/test/resources/GET.feature:15]: That was an example of how we are supposed to write an escenario, I can put it back if you want

Comment (ramiroAT) [reply src/test/resources/GET.feature:15]: Got it. Then is ok you have removed it from the file.

**Review by ramiroAT — APPROVED** [review]

**Review by RodrigoAgileThought — APPROVED** [review]
